opennms-doc/guide-admin/src/asciidoc/text/dnsresolver/introduction.adoc

```diff
@@ -0,0 +1,38 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+In order to customize the DNS servers that are queried, the following commands can be used:
```

Comment:
Can we add a note to clarify that this is currently only used by `telemetryd`? (Not so obvious now that it's moved to its own section.)

Reply:
Done